### DISCUSSION TITLE
Can't start wiki.js 2.0.1 with new installation

### DISCUSSION THREAD

**Discussion by brunelloalessio** [requarks/wiki#3345] [Help / Questions]
**Question**
I can't start node server for wiki.js, node server logs stops to:
"Syncing locales with Graph endpoint" OR with "Rebuilding page tree" without errors

**Host Info (please complete the following information):**
OS: [CentOs]
Wiki.js version: [2.0.1]
Database engine: [mariadb]

**Comment by NGPixel** [comment]
Please post the full logs. I can't help you with that much info.
